src/components/views/Groups.js

```diff
@@ -1,16 +1,113 @@
 import React from 'react'
 import Group from '../cards/Group'
 import ContentWrapper from '../../styles/ContentWrapper'
+import { graphql, gql, withApollo } from 'react-apollo'
+import Button from '../shared/Button'
+import Alert from '../shared/Alert'
+import ActionSlide from '../shared/ActionSlide'
+
 class Groups extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      inTransition: false,
+      openMenu: false,
+      active: false
+    }
+    this.handleButtonClick = this.handleButtonClick.bind(this)
+    this.handleNewGroup = this.handleNewGroup.bind(this)
+  }
+  handleButtonClick() {
+    this.setState({
+      openMenu: !this.state.openMenu,
+      groups: []
+    })
+  }
+
+  handleNewGroup(e, refs) {
+    e.preventDefault()
+    this.props.client
```

Comment:
Should this be handled by a mutation injected as a prop instead of `props.client.mutate()`?